api/resources/translations/messages-fr.properties

```diff
@@ -1264,7 +1264,10 @@ training_cards.confirm.title = Quitter l'entraînement?
 training_cards.error.loading = Erreur lors du chargement de la formation. Veuillez contacter votre superviseur.
 training_cards.error.save = Erreur lors de l'enregistrement de la formation.
 training_cards.form.saved = Formation terminée.
-training_cards.modal.title = Changements importants
+training_materials.page.no_more_trainings = Plus de formations
+training_materials.page.no_selected = Aucun matériel de formation sélectionné
+training_materials.page.no_trainings = Aucune formation trouvée
+training_materials.page.title = Matériel de formation
```

Comment:
@andrablaj, is this good french?

Reply:
I added a suggestion and tagged @m5r. All the rest sound good to me.

Reply:
+1 all the rest sound good! :+1:

Reply:
Thank you!